x-pack/plugins/uptime/public/components/overview/monitor_list/monitor_list.tsx

```diff
@@ -190,31 +187,7 @@ export const MonitorListComponent: React.FC<Props> = ({
 
   return (
     <EuiPanel>
-      <EuiFlexGroup>
-        <EuiFlexItem>
-          <EuiTitle size="xs">
-            <h5>
-              <FormattedMessage
-                id="xpack.uptime.monitorList.monitoringStatusTitle"
-                defaultMessage="Monitor status"
-              />
-            </h5>
-          </EuiTitle>
-        </EuiFlexItem>
-        <EuiFlexItem grow={false}>
-          <EuiTitle size="xs">
-            <h5>
-              <Link to={CERTIFICATES_ROUTE} data-test-subj="uptimeCertificatesLink">
-                <FormattedMessage
-                  id="xpack.uptime.monitorList.viewCertificateTitle"
-                  defaultMessage="View certificates status"
-                />
-              </Link>
-            </h5>
-          </EuiTitle>
-        </EuiFlexItem>
-      </EuiFlexGroup>
-
+      <MonitorListHeader />
```

Comment:
We already have `statusFilter` here, so we should pass that as a prop here IMHO, to prevent the use of the hook in the filter button.

Reply:
it's going two components down, so declaring props interfaces on two components, so i feel like calling hook is the cleanest way.